keps/sig-storage/20190129-csi-pod-info-on-mount.md

```diff
@@ -60,3 +60,5 @@ This KEP contains details that are missing from the design proposal.
 ## Implementation History
 
 * K8s 1.12: Alpha implementation
+* K8s 1.14: Beta implementation
+* K8s 1.17: GA implementation
```

Comment:
Do we also want to enhance this feature to also provide PVC metadata during provisioning?

Reply:
IMO, we could use existing "env. variable" expansion introduced  in https://github.com/kubernetes-csi/external-provisioner/pull/69